File: src/com/eagle/programmar/Bash/Terminals/Bash_Literal.java
// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, Aug 6, 2011

package com.eagle.programmar.Bash.Terminals;

import com.eagle.tokens.terminals.TerminalLiteralToken;

public class Bash_Literal extends TerminalLiteralToken
{
	public Bash_Literal()
	{
		super("'\"`", true, '\\', false, false);
	}
}

File: csrc/com/eagle/programmar/Bash/Terminals/Bash_Literal.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, Aug 6, 2011

namespace com.eagle.programmar.Bash.Terminals
{
	using TerminalLiteralToken = com.eagle.tokens.terminals.TerminalLiteralToken;

	public class Bash_Literal : TerminalLiteralToken
	{
		public Bash_Literal() : base("'\"`", true, '\\', false, false)
		{
		}
	}

}
